### ISSUE TITLE
TaskDialog Demo

### ISSUE THREAD

**Issue by PeterPanino** [EtheaDev/StyledComponents#13]
This is the TaskDialog Demo at run-time:

![image](https://github.com/EtheaDev/StyledComponents/assets/5140376/618a4049-92ea-4c7d-b62f-0202edbe2791)

Can you please provide a better layout? Thank you for the **StyledComponents** library - you are great!

**Comment by carloBarazzetta** [comment]
try to remove those lines from Loaded method and give me a feedback:
```
procedure TMainForm.Loaded;
begin
  //Font.Assign(Screen.IconFont);
  //Screen.MessageFont.Assign(Font);
  inherited;
end;

```

**Comment by carloBarazzetta** [comment]
Fixed in last 3.3.0 version
